.github/workflows/ci-build.yml

```diff
@@ -16,22 +16,22 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: volta-cli/action@v4
-      - run: yarn install --frozen-lockfile --ignore-engines
-      - run: yarn lint
-      - run: yarn test
+      - uses: actions/checkout@v3
+      - uses: NullVoxPopuli/action-setup-pnpm@v1
+      - run: pnpm lint
+      - run: pnpm test
 
   floating-dependencies:
     timeout-minutes: 10
     name: 'Floating Dependencies'
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: volta-cli/action@v4
-      - run: yarn install --no-lockfile
-      - run: yarn test
+      - uses: actions/checkout@v3
+      - uses: NullVoxPopuli/action-setup-pnpm@v1
```

Comment:
abstracting away `--no-lockfile` is not the point of the action, but to abstract away all the menial stuff needed for a pnpm repo. It's always possible to be more flexible if you don't use an abstraction :sweat_smile:

Reply:
I'm not sure I'm following. The purpose of the action seems to be to make setting up pnpm easier, but why must it already do the install itself? If you just have users run `pnpm install --no-lockfile` themselves, you don't need the `no-lockfile` option, and reading this CI file becomes clearer because you can read and see what's happening.

Reply:
> but why must it already do the install itself?

reduced boilerplate

> and reading this CI file becomes clearer because 

everything is clearer when you don't need to read anything unfamiliar :thinking: 


I understand your concerns, I do, but as a maintainer of _many_ addons, I find boilerplate in these config files immensely annoying. 
I also want to make actions that abstract away the setup and such for ember-try and typescript version matrix testing.
IMO, having too much verbosity is about as harmful as too little. Thankfully, when errors occur in these actions, it's clear where, due to the stack trace being totally transparent about what code is where / from what dependency / action / etc.